util/chplenv/chpl_3p_jemalloc_configs.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import third_party_utils, utils
+from utils import memoize
+
+
+@memoize
+def get_uniq_cfg_path():
+    return third_party_utils.default_uniq_cfg_path()
+
+@memoize
+def get_link_args():
+    # -lpthread required to satisfy "undefined reference to `pthread_atfork"
+    # and others. Note that this only appears to be needed because we build
+    # jemalloc statically. When using a .so the -lpthread is not required.
+    # @gbt any idea why that is/anything to change or is this ok?
+    return third_party_utils.default_get_link_args('jemalloc',
+                                                   libs=['-ljemalloc', '-lpthread'])
```

Comment:
Wait, don't think this is right.  The libs argument to default_get_link_args() is the .la file(s) to look at. It should be (probably) libjemalloc.la.  ("la" filename extension, not "a")   Unless jemalloc doesn't use libtool and therefore doesn't create a .la file.  Then you'd have to name the libs explicitly.  Not having a .la file would also explain why libpthread isn't getting added automagically, I think.